src/modules/selection/SelectionProvider.jsx

```diff
@@ -42,26 +34,15 @@ const SelectionProvider = ({ children }) => {
   const location = useLocation()
   const [selectedItems, setSelectedItems] = useState({})
   const [isSelectionBarOpen, setSelectionBarOpen] = useState(false)
-  const [focusedIndex, setFocusedIndex] = useState(0)
-  const [lastSelectedIndex, setLastSelectedIndex] = useState(null)
-  const [isKeyboardNavigating, setIsKeyboardNavigating] = useState(false)
-  const itemsListRef = useRef([])
+  const [isSelectAll, setIsSelectAll] = useState(false)
```

Comment:
nit: this add is not part of the extraction refactor, it's a fix/feat isn't it? It should be in an other commit